tests/test_p2p_daemon.py

```diff
@@ -110,6 +114,13 @@ def handle_add_torch(args):
     return serialize_torch_tensor(result).SerializeToString()
 
 
+def handle_add_torch_with_exc(args):
+    try:
+        return handle_add_torch(args)
+    except:
```

Comment:
```suggestion
    except Exception:
```

Catching everything is a bad practice in Python (e.g. it may disrupt the work of Ctrl+C which raises `KeyboardInterrupt`).